README.adoc

```diff
@@ -55,7 +55,7 @@ The script to create a pipeline is included with the Kabanero Pipelines project.
 Go to the `kabanero-pipelines` folder that you recently cloned. Find the `pipelines/incubator/build-deploy-pipeline.yaml` file, and remove the section of the file that is highlighted:
 
 // image::/img/guide/test-build-pipeline.png[link="/img/guide/test-build-pipeline.png" alt=""]
-[source,yaml,highlight=81..90]
+[source,yaml,linenums,highlight=81..90]
```

Comment:
I would suggest just saying "remove the below section from the file"

then only putting the section the reader should delete in the code block. Relying only on the highlighting probably isn't 100% accessible.